stock_picking_batch_validate_confirm/wizards/batch_confirm.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import fields, models
+
+
+class BatchConfirm(models.TransientModel):
+    _name = "batch.confirm"
```

Comment:
suggestion: "batch" is a pretty wide term. I'd use a more pedantic name for the wizard:

```suggestion
class StockPickingBatchConfirm(models.TransientModel):
    _name = "stock.picking.batch.confirm"
```